core/src/com/unciv/ui/worldscreen/Minimap.kt

```diff
@@ -17,13 +18,15 @@ import com.unciv.ui.utils.IconCircleGroup
 import com.unciv.ui.utils.ImageGetter
 import com.unciv.ui.utils.onClick
 import com.unciv.ui.utils.surroundWithCircle
+import java.util.*
+import kotlin.collections.HashMap
 import kotlin.math.max
 import kotlin.math.min
 
 class Minimap(val mapHolder: WorldMapHolder) : Table(){
     private val allTiles = Group()
     private val tileImages = HashMap<TileInfo, Image>()
-    private val scrollPosistionIndicator = ImageGetter.getImage("OtherIcons/Camera")
+    private val scrollPositionIndicator = ImageGetter.getImage("OtherIcons/Camera")
```

Comment:
Absolutely fix spelling errors that you encounter in the future :D
Part of the basic code sanitation is easy-to-read variables, and spelling mistakes reduce that readability